roles/nvidiadl_ssd_benchmark_deploy/files/nvidia-pod.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidiadl
+  labels:
+    name: nvidiadl
+spec:
+  restartPolicy: Never   
+  containers:
+  - name: nvidiadl
+    image: quay.io/openshift-psap/nvidiadl-ssd-training-benchmark
+    resources:
+      limits:
+        nvidia.com/gpu: 1
+      requests:
+        nvidia.com/gpu: 1
+    volumeMounts: 
+      - name: config-volume
+        mountPath: /mnt/entrypoint/entrypoint.sh
+        readOnly: false
+        subPath: entrypoint.sh
+      - mountPath: /storage/
+        name: storage
+    command:
+      - bash
+      - -c
+      - "/mnt/entrypoint/entrypoint.sh"
+  initContainers:
+  - name: init-fetch-data
+    image: quay.io/openshift-psap/nvidiadl-ssd-training-benchmark
+    command:
+      - bash
+      - -c
+      - "/workspace/download_dataset.sh '/storage'"
```

Comment:
I assume this script won't download from our mirror, 
you can already create another entry in the CM with a custom script, which will be modified once everything is setup with our mirror (use the code from the current `download_dataset,sh`file meanwhile)